<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ajout</class>
 <widget class="QDialog" name="Ajout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>453</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupFileChoice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>731</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Choix du fichier</string>
    </property>
    <widget class="QPushButton" name="buttonParcourir">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Parcourir</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textFile">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>541</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Choisissez un fichier</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupKeywords">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>731</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Mots clés</string>
    </property>
    <widget class="QListWidget" name="listAll">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>256</width>
       <height>191</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>wallpaper</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HQ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BEST</string>
      </property>
     </item>
    </widget>
    <widget class="QListWidget" name="listDoc">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAjouter">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter &gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSupprimer">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer &lt;&lt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tout les mots clés</string>
     </property>
    </widget>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mots clés du document</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonNouveau">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>200</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouveau</string>
     </property>
    </widget>
    <zorder>listAll</zorder>
    <zorder>listDoc</zorder>
    <zorder>buttonAjouter</zorder>
    <zorder>buttonSupprimer</zorder>
    <zorder>label1</zorder>
    <zorder>label2</zorder>
    <zorder>buttonNouveau</zorder>
    <zorder>groupFileChoice</zorder>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>156</width>
      <height>23</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
